nuxeo/values.yaml

```diff
@@ -61,6 +60,7 @@ nuxeo:
     annotations: {}
   customParams: |-
     # Custom parameters to be appended to nuxeo.conf
+    elasticsearch.indexNumberOfReplicas=0
```

Comment:
See my comment https://github.com/nuxeo/nuxeo-helm-chart/pull/10#discussion_r406050761, not sure we can do that.

Reply:
I've been trying this morning but couldn't find a way yet